lib/maid/watch.rb

```diff
@@ -6,7 +6,14 @@ class Maid::Watch
 
   def initialize(maid, path, options = {}, &rules)
     @maid = maid
-    @options = options
+    if options.nil? 
+      @lazy = true
+      @options = { wait_for_delay: 10, 
+                   ignore: [/\.crdownload$/, /\.download$/, /\.aria2$/, /\.td$/, /\.td.cfg$/, /\.part$/] }
```

Comment:
@benjaminoakes Yes, it should. But I can't find a proper way to do so. The `ignore` option can only take paths or regular expressions while `downloading?` is more complexed. Any suggestions?